src/Microsoft.Azure.WebJobs.Host/Blobs/Triggers/BlobTriggerBinding.cs

```diff
@@ -266,10 +266,12 @@ public Task<IListener> CreateListenerAsync(ListenerFactoryContext context)
             }
 
             IStorageBlobContainer container = _blobClient.GetContainerReference(_path.ContainerNamePattern);
+            // blobtriggerbinding uses dispatch
```

Comment:
nit: remove unnecessary comment. Also the local declaration naming/casing is off.